.github/actions/test-dotnet/action.yml

```diff
@@ -38,9 +38,13 @@ runs:
       run: docker ps
       shell: bash
     
-    - name: debug
-      run: ls -la
-      shell: bash
+    - name: Load .env file
+      uses: aarcangeli/load-dotenv@v1.0.0
```

Comment:
fwiw that action does not appear to be maintained or well-used

what about https://github.com/marketplace/actions/simple-dotenv ? 

Or can we do this ourselves and avoid bringing in another dependency?